packages/perseus/src/widgets/interactive-graphs/graphs/components/use-control-point.tsx

```diff
@@ -76,6 +83,15 @@ export function useControlPoint(params: Params): Return {
         constrainKeyboardMovement: constrain,
     });
 
+    // if custom aria label is not provided, will use default of sequence number and point coordinates
+    const pointAriaLabel =
+        ariaLabel ||
+        strings.srPointAtCoordinates({
+            num: sequenceNumber,
```

Comment:
From my comment above, maybe this line could be

```
num: sequenceNumber ?? 1,
```

Reply:
Defaulted the prop in the de-structuring section